spec/factories/product.rb

```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :product do
+    digit = Faker::Number.between(1, 10)
+
+    code { Faker::Number.number(digit) }
+    name { Faker::Name.name }
+    description { Faker::Lorem.paragraph }
+    value { Faker::Number.decimal(digit, 2) }
```

Comment:
You don't need line 3, just use Faker::Number.positive.round(2) on line 8 and it's done.